helper/schema/schema.go

```diff
@@ -62,10 +62,20 @@ type Schema struct {
 	DiffSuppressFunc SchemaDiffSuppressFunc
 
 	// If this is non-nil, then this will be a default value that is used
-	// when this item is not set in the configuration/state.
+	// when this item is not set in the configuration.
 	//
-	// DefaultFunc can be specified if you want a dynamic default value.
-	// Only one of Default or DefaultFunc can be set.
+	// DefaultFunc can be specified if you want a computed default value.
```

Comment:
I'm not much of a grammarian, but I think the second person here is awkward. Can we make `DefaultFunc` active?   `DefaultFunc can be specified to compute a default value`.